lib/sandbox.nix

```diff
@@ -9,6 +9,7 @@
   name ? "emacs-sandboxed",
   userEmacsDirectory ? null,
   extraBubblewrapOptions ? [],
+  enableWindow ? true,
```

Comment:
At first sight, I was not sure what this option would mean. It would be better to name it `enableWindowSystem`, `windowSystem`, `enableX`, `withX`, etc.